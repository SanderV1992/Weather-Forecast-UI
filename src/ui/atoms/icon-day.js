import React from 'react'
import PropTypes from 'prop-types'

import { Icon } from 'semantic-ui-react'


export const IconDay = ({ name, size }) => (
  name === 'DAY'
    ? <Icon name="sun" size={size} color="yellow" />
    : <Icon name="moon" size={size} />
)

IconDay.propTypes = {
  name: PropTypes.string,
  size: PropTypes.string,
}

IconDay.defaultProps = {
  name: 'DAY',
  size: 'large',
}
