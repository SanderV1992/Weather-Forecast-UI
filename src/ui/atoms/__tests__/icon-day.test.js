import React from 'react'
import expect from 'expect'
import { shallow } from 'enzyme'

import { IconDay } from '../icon-day'


describe('<IconDay />', () => {

  test('should render with default props', () => {
    const icon = shallow(<IconDay />)

    expect(icon).toBeDefined()
    expect(icon).toMatchSnapshot()
  })

})
