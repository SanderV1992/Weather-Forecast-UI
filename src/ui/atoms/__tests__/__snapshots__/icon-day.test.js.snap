// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconDay /> should render with default props 1`] = `ShallowWrapper {}`;
